### PR TITLE
Updated DevConf 2016 blog entry

### PR DESCRIPTION
**Pull request by mykaul**

### CODE REVIEW

**Comment by mykaul** [comment]
@doron-fediuck  - fixed according to your comments.
@thatdocslady  - would be happy to get your review as well.

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: My Devconf.CZ 2016 experience
+author: ykaul
+tags: community, news, blog
+date: 2016-03-02 15:00:00 CET
+---
+
+On the first weekend of February I've had the pleasure of attending DevConf.CZ 2016, which took place in the wonderful city of Brno, CZ.
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: * I've had the pleasure > I had the pleasure
* Brno, CZ. > Brno, Czech Republic.

**Comment by thatdocslady** [comment]
General request: Use markdown format for links where [Link Text](URL). Currently the whole list of links is at the bottom of the blog post, which is more appropriate for mailing lists.

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Attended [in-depth look of virtual machine migration algorithms - Marcelo Tosatti](https://www.youtube.com/watch?v=XkMIMJKJeTY "in-depth look of virtual machine migration algorithms - Marcelo Tosatti"). I hope to see some of it in coming versions of QEMU, while we work on other improvements to this critical feature.
+- [Growing the ARM server ecosystem - Jim Perrin](https://www.youtube.com/watch?v=q4gU87wdRtI "Growing the ARM server ecosystem - Jim Perrin") was interesting. Seems like a great positive effort, but the road is still long for complete support. Would be cool to support ARM based hosts running virtual machines!
+- Attended [Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty](https://www.youtube.com/watch?v=Jx93riUF5_I "Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty"). The results were not conclusive, though some improvements in native brought it close to or better in some cases to threads. Seems that our default of heuristic for threads in file-based storage and native in block based storage is fine for the time being, but we'll watch closely for developments in this area.
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Edits:

* brought it close to or better in some cases to threads > brough it close to, or in some cases better than threads

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Attended [Dockerizing JBoss Products - David Becvarik](https://www.youtube.com/watch?v=NpyEoFlDzOQ "Dockerizing JBoss Products - David Becvarik"). Great work done by the JBoss team to improve and streamline packaging of JBoss into containers.
+- Attended [Upstream First Testing - Tim Flink](https://www.youtube.com/watch?v=15-yXOJuonQ "Upstream First Testing - Tim Flink"). I discussed with the presenter the possibility to run oVirt with [Lago project on GitHub](https://github.com/lago-project/lago "Lago project on GitHub") in Fedora, in a CI way, by monitoring distgit changes for relevant packages (such as lvm2, device-mapper*, libvirt and others.
+- Obviously attended [High performance VMs in OpenStack - Nikola Dipanov](https://www.youtube.com/watch?v=9J_sEdlpIlQ "High performance VMs in OpenStack - Nikola Dipanov") - looks like oVirt is already doing a lot of what OpenStack are working to achieve high-performance from KVM!
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: * Please remove "obviously", not needed.
* "what OpenStack are working" > "what OpenStack is doing"

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- Attended [Dockerizing JBoss Products - David Becvarik](https://www.youtube.com/watch?v=NpyEoFlDzOQ "Dockerizing JBoss Products - David Becvarik"). Great work done by the JBoss team to improve and streamline packaging of JBoss into containers.
+- Attended [Upstream First Testing - Tim Flink](https://www.youtube.com/watch?v=15-yXOJuonQ "Upstream First Testing - Tim Flink"). I discussed with the presenter the possibility to run oVirt with [Lago project on GitHub](https://github.com/lago-project/lago "Lago project on GitHub") in Fedora, in a CI way, by monitoring distgit changes for relevant packages (such as lvm2, device-mapper*, libvirt and others.
+- Attended [High performance VMs in OpenStack - Nikola Dipanov](https://www.youtube.com/watch?v=9J_sEdlpIlQ "High performance VMs in OpenStack - Nikola Dipanov") - looks like oVirt is already doing a lot of what OpenStack is working on to achieve high-performance from KVM!
+- Preached on running Lago with OpenStack, Gluster and Cockpit. At least the first two items look really promising and I'm looking forward to a collaboration in those efforts. I might just try to do the Gluster one myself.
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Not sure what "preached" means in this context, but you might want to select a word with less of an evangelist association :)

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- [Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad](https://www.youtube.com/watch?v=rqavfmPAt7o "Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad") was interesting for me, as Avocado seems like a cool, mature testing framework - which I hope we can use in the future in CI testing.
+- Having used the [oVirt Live](http://www.ovirt.org/download/ovirt-live/"http://www.ovirt.org/download/ovirt-live/") USB DoK image in both FOSDEM and DevConf, I've found several items where we can improve in and filed relevant RFEs for them [1][2].
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Please rework the links into the sentence like you did with the others, so you can for example write "filed RFE <number> and <number2>".

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+[Smart VM Scheduling - Martin Sivák](https://www.youtube.com/watch?v=cQqJEiK7-Ug "Smart VM Scheduling - Martin Sivák")
+[Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina](https://www.youtube.com/watch?v=V1JQtmdleaM "Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina") ,
+[Ceph integration with oVirt using Cinder - Nir Soffer](https://www.youtube.com/watch?v=4CbHTAkVDZo "Ceph integration with oVirt using Cinder - Nir Soffer").
+- The Gluster team has additionally provided more presentations:
+[oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu](https://www.youtube.com/watch?v=XudYwEWQF7U "oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu") ,
+[Improvements in gluster for virtualization use case - Prasanna Kumar Kalever](https://www.youtube.com/watch?v=TczVVCbm8NE "Improvements in gluster for virtualization use case - Prasanna Kumar Kalever"), which were all well received.
+
+I'd like to thank our project's community managers Mikey and Brian for once again, just one week after FOSDEM, leading the effort of representing the oVirt project and community in this event.
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: "our project's currenty community manager Mikey Ariel, and former community manager Brian Proffitt"

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+Happy to hear any feedback,
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Please remove this line, not needed for the blog post

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Yaniv Kaul
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Please remove this line, not needed for the blog post

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+1.[[RFE] register .vv files so they'll be opened automatically with remote-viewer](https://bugzilla.redhat.com/show_bug.cgi?id=1307261 "[RFE] register .vv files so they'll be opened automatically with remote-viewer")
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: Please rework these link refs into the actual body (see line note above)

**Comment by thatdocslady** [comment]
@mykaul - sorry for the staggered comments, please update the PR (and also change the date to today's date) and I'll merge it and publish :) thanks again for submitting the post, it's great!

File: source/blog/2016-02-24-devconf-2016.html.md (hunk 6 above)
Comment (mykaul) [inline source/blog/2016-02-24-devconf-2016.html.md]: Fixed.

File: source/blog/2016-02-24-devconf-2016.html.md (hunk 5 above)
Comment (mykaul) [inline source/blog/2016-02-24-devconf-2016.html.md]: I prefer not to - those are OK as is.

File: source/blog/2016-02-24-devconf-2016.html.md (hunk 4 above)
Comment (mykaul) [inline source/blog/2016-02-24-devconf-2016.html.md]: 'evangelized' is better? ;-)

(I did exactly that).

File: source/blog/2016-02-24-devconf-2016.html.md (hunk 7 above)
Comment (mykaul) [inline source/blog/2016-02-24-devconf-2016.html.md]: Fixed.

File: source/blog/2016-02-24-devconf-2016.html.md (hunk 8 above)
Comment (mykaul) [inline source/blog/2016-02-24-devconf-2016.html.md]: Fixed.

**Comment by mykaul** [comment]
@thatdocslady - it's now, or we wait for FOSDEM 17'  ;-)

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
+---
+title: My Devconf.CZ 2016 experience
+author: ykaul
+tags: community, news, blog
+date: 2016-03-07 15:00:00 CET
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: please change to today's date

File: source/blog/2016-02-24-devconf-2016.html.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+[Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina](https://www.youtube.com/watch?v=V1JQtmdleaM "Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina") ,
+[Ceph integration with oVirt using Cinder - Nir Soffer](https://www.youtube.com/watch?v=4CbHTAkVDZo "Ceph integration with oVirt using Cinder - Nir Soffer").
+- The Gluster team has additionally provided more presentations:
+[oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu](https://www.youtube.com/watch?v=XudYwEWQF7U "oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu") ,
+[Improvements in gluster for virtualization use case - Prasanna Kumar Kalever](https://www.youtube.com/watch?v=TczVVCbm8NE "Improvements in gluster for virtualization use case - Prasanna Kumar Kalever"), which were all well received.
+
+I'd like to thank our project's currently community manager Mikey Ariel, and former community manager Brian Proffitt" for once again, just one week after FOSDEM, leading the effort of representing the oVirt project and community in this event.
```
Comment (thatdocslady) [inline source/blog/2016-02-24-devconf-2016.html.md]: * currently > current
* remove " after Proffitt

**Comment by thatdocslady** [comment]
@mykaul two last tiny comments and then I'll merge and publish it today :) thanks again!

**Comment by mykaul** [comment]
@thatdocslady  - please... pretty please...

**Pull request merged**